Xamarin.Forms.Controls/GalleryPages/OSThemesGalleryXaml.xaml

```diff
@@ -0,0 +1,55 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms" xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" xmlns:gallerypages="clr-namespace:Xamarin.Forms.Controls.GalleryPages" x:Class="Xamarin.Forms.Controls.GalleryPages.OSThemesGalleryXaml">
+    <ContentPage.Resources>
+        <AppThemeColor x:Key="MyColor" Light="HotPink" Dark="Yellow" />
+        <Style x:Key="OSThemeStyle" TargetType="Label" >
+            <Setter Property="TextColor" Value="{OnAppTheme Black, Light=Green, Dark=Red}" />
+        </Style>
+        <StyleSheet>
+            <StyleSheetExtension.Style>
+                <OnAppTheme x:TypeArguments="x:String">
+                    <OnAppTheme.Light>
+                        <![CDATA[
+                            #cssStyledLabel {
+                                color: purple;
+                            }
+                        ]]>
+                    </OnAppTheme.Light>
+                    <OnAppTheme.Dark>
+                        <![CDATA[
+                            #cssStyledLabel {
+                                color: orange;
+                            }
+                        ]]>
+                    </OnAppTheme.Dark>
+                </OnAppTheme>
+            </StyleSheetExtension.Style>
+        </StyleSheet>
+    </ContentPage.Resources>
+    <ScrollView>
+        <StackLayout HorizontalOptions="Center" VerticalOptions="Center">
+            <Label TextColor="{OnAppTheme Light=Green, Dark=Red}">This text is green or red depending on Light (or default) or Dark</Label>
+            <Label>
+                <Label.TextColor>
+                    <OnAppTheme x:DataType="Color" Light="Green" Dark="Red" />
+                </Label.TextColor>
+            </Label>
+            <Label Style="{DynamicResource Key=OSThemeStyle}">This text is green or red depending on Light (or default) or Dark (through style)</Label>
+            <Label TextColor="{DynamicResource MyColor}">This text is HotPink or Yellow depending on Light (or default) or Dark (through AppThemeColor)</Label>
+            <Label TextColor="{StaticResource MyColor}">This text is HotPink or Yellow depending on Light (or default) or Dark (through AppThemeColor)</Label>
+            <Label x:Name="cssStyledLabel">This text is Purple or Orange depending on Light (or default) or Dark (through CSS)</Label>
+            <Label>The image below is a Xamarin logo or fruits depending on Light (or default) or Dark</Label>
+            <Image Source="{OnAppTheme Light=xamarinlogo, Dark=Fruits.jpg}" />
```

Comment:
It is not important, but the xamarinlogo image will not work in UWP (path).

Reply:
Fixed!